lib/MusicBrainz/Server/Controller/OtherLookup.pm

```diff
@@ -6,6 +6,26 @@ use Moose::Util qw( find_meta );
 use MusicBrainz::Server::Translation qw( l );
 use MusicBrainz::Server::Constants qw( entities_with );
 
+sub quoted_query {
```

Comment:
Please use the term _phrase_ as in Lucene documentation: https://lucene.apache.org/core/7_7_2/queryparser/org/apache/lucene/queryparser/classic/package-summary.html#Terms